### ISSUE TITLE
getBlksMaybeFromNet timeout

### ISSUE THREAD

**Issue by ognots** [filecoin-project/go-filecoin#1941]
### Description

The chain syncer has a timeout of 1 second to process blocks

Determine a more robust timeout.

Encountered error in logs:
```
ERROR       node: processBlock(): processing block from network: context deadline exceeded node.go:1096
```

### Acceptance criteria

processing a block should have a more realistic deadline

### Risks + pitfalls

There's a lock further up the call stack 
https://github.com/filecoin-project/go-filecoin/blob/57d93747c35c88e6f25af2d36c821201bb7cfc9c/chain/default_syncer.go#L324-L325

### Where to begin

https://github.com/filecoin-project/go-filecoin/blob/57d93747c35c88e6f25af2d36c821201bb7cfc9c/chain/default_syncer.go#L81-L113

**Comment by ZenGround0** [comment]
#2065

**Comment by anorth** [comment]
@acruikshank I think you adjusted a timeout like this recently - is this issue now resolved?

**Comment by acruikshank** [comment]
Yes. This has been addressed in #2425 in response to a duplicate issue #2424.

**Comment by ognots** [comment]
Closed by https://github.com/filecoin-project/go-filecoin/pull/2425
